app/test/integration-test/pom.xml

```diff
@@ -199,6 +199,13 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+      <!-- Required only for CircleCi build as the used JVM to run the tests has limited cryptography -->
+      <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```

Comment:
Could we try switching to [adoptopenjdk/openjdk8](https://hub.docker.com/r/adoptopenjdk/openjdk8) image in place of `openjdk:8-jdk` instead.

Reply:
as I am using Testcontainers in the tests I need to use the machine executor in CircleCi for this job. And this is using a ubuntu vm image with some JDK in it that has this cryptography restrictions. So not sure how to adjust this used JDK in this virtual machine.

Reply:
Perhaps we could install the [AdoptOpenJDK debian package](https://adoptopenjdk.net/installation.html#linux-pkg)?

Reply:
Let me merge this one and tackle the cryptography afterwards so the hourly failing builds disappear for now.